recipe/meta.yaml

```diff
@@ -114,6 +113,13 @@ outputs:
     {% set tests_to_skip = tests_to_skip + " or test_overrides" %}  # [blas_impl == 'mkl']
     # https://github.com/numpy/numpy/issues/15243
     {% set tests_to_skip = tests_to_skip + " or test_loss_of_precision" %}  # [s390x or ppc64le]
+    # https://github.com/numpy/numpy/issues/3858 - could be related
+    {% set tests_to_skip = tests_to_skip + " or test_big_arrays" %}  # [s390x or ppc64le]
```

Comment:
Which error message did you get? If you still have it, could you add it to the PR description?
Another issue seems to show that this tests uses a fair amount of memory. It could be the reason. https://github.com/numpy/numpy/issues/20125

Reply:
there was not error message at all the whole test suite just crushed not giving any info about the affected test, I have only find out which one could be by running all linux-64 and comparing log outputs

Reply:
I'll try to reproduce and see if I find a bit more information.